src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -73,58 +73,58 @@
     /** The number of PF reads that are not marked as duplicates. */
     public long PF_UNIQUE_READS;
 
-    /** PF reads / total reads.  The percent of reads passing filter. */
+    /** The fraction of reads passing the vendor's filter (PF reads/total reads).   */
     public double PCT_PF_READS;
 
-    /** PF Unique Reads / Total Reads. */
+    /** The fraction of unique reads from the total number of PF-reads [PF (Unique reads/Total reads)]. */
     public double PCT_PF_UQ_READS;
 
-    /** The number of PF unique reads that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF-unique reads that aligned to the reference genome with a mapping score > 0. */
     public long PF_UQ_READS_ALIGNED;
 
-    /** PF Reads Aligned / PF Reads. */
+    /** The fraction of PF reads aligned to the reference genome from the total number of PF reads. */
     public double PCT_PF_UQ_READS_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF unique bases that are aligned to the reference genome with mapping scores > 0. */
     public long PF_BASES_ALIGNED;
 
-    /** The number of bases in the PF aligned reads that are mapped to a reference base. Accounts for clipping and gaps. */
+    /** The number of bases in the PF aligned reads that map to a reference base. Accounts for clipping and gaps. */
     public long PF_UQ_BASES_ALIGNED;
 
-    /** The number of PF aligned bases that mapped to a baited region of the genome. */
+    /** The number of PF aligned bases that map to the bait regions of the genome. */
     public long ON_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to within a fixed interval of a baited region, but not on a baited region. */
+    /** The number of PF aligned bases that map to within a fixed interval containing a baited region, but not within the baited section per se. */
     public long NEAR_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to neither on or near a bait. */
+    /** The number of PF aligned bases that map distal to a baited region. */
     public long OFF_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to a targeted region of the genome. */
+    /** The number of PF aligned bases that map to a hybrid selection target region of the genome. */
```

Comment:
"The targeted" is right. Hybrid selection is one type of targeting and this metric is for TargetedPCR.